api/v1/helpers.go

```diff
@@ -44,13 +46,15 @@ func (backend Backend) Location() (string, error) {
 	if backend.S3 != nil {
 		return "s3:" + backend.S3.Bucket, nil
 	} else if backend.GCS != nil {
-		return "gs:" + backend.GCS.Bucket, nil
+		return "gcs:" + backend.GCS.Bucket, nil
```

Comment:
It will be `gs`. It directly comes from Restic. Ref: https://restic.readthedocs.io/en/stable/030_preparing_a_new_repo.html